### DISCUSSION TITLE
alignment of image and text

### DISCUSSION THREAD

**Discussion by sparatko242** [imfing/hextra#222] [Q&A]
please, 

I have several images with text in page, but each image is taking all width and text is under it.

```
![](image.png)
longer text about image
```

I would like to align image with some size (e.g. 150px) to left and text to the right side of image. Is it possible?
or I could used cards but text is to long for subtitle.

thanks

**Comment by imfing** [comment]
Please use HMTL [img](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/img) tag, this as an example:

https://gist.github.com/DavidWells/7d2e0e1bc78f4ac59a123ddf8b74932d
